src/index.js

```diff
@@ -146,6 +149,10 @@ function truthy(str) {
   return _.includes(['TRUE', 'YES', 'T', 'Y'], normalizedStr);
 }
 
+function closeCovidBanner() {
+  document.getElementById('covid-banner').style.display = "none"
```

Comment:
I'm not an a11y expert by any means, but would it be worth adding `.setAttribute("aria-hidden", true);` for this?

Reply:
I also don't know too too much but did a little looking and that seems reasonable, yes! Will add it!